otel/README.md

```diff
@@ -54,7 +54,7 @@ service:
 
 For further information on the Datadog exporter settings and how to configure the pipeline, see [Datadog exporter for OpenTelemetry Collector][3].
 
-See the [Metrics section](#metrics) for a list of metrics that are provided by this check. If you're using the `hostmetrics` receiver as in the sample configuration above. You can send arbitrary metrics with other OpenTelemetry Collector components.
+See the [Metrics section][5] for a list of metrics that are provided by this check. If you're using the `hostmetrics` receiver as in the sample configuration above. You can send arbitrary metrics with other OpenTelemetry Collector components.
```

Comment:
Hi Kari, 
Could you please change
See the [Metrics section][5] for a list of metrics that are provided by this check.
into
See the [Metrics section][5] for metrics types and [Metadata.csv][8] for a list of metrics provided by this check.